README.md

```diff
@@ -49,10 +49,16 @@ There are a couple of session-based open source libraries that are nice alternat
 
 You can [sign up for a free account](https://app.posthog.com/signup) on our hosted platform.
 
-## Deploy to Heroku
+## Deploy to PaaS
 
+### Heroku Install
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy?template=https://github.com/posthog/posthog)
 
+### Platform.sh Install
```

Comment:
We list all of our alternative ways of deploying [here](https://posthog.com/docs/deployment), which you can [edit here](https://github.com/PostHog/posthog.com/blob/master/contents/docs/deployment.md).

Reply:
Ok, removed the reference from the main README file and put more detailed instructions in a .platform/README.md file, and contributed to the other docs.